flathub.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
these are the only arches. This file is not needed.

Reply:
RIP 32bit 😢